main.css

```diff
@@ -0,0 +1,46 @@
+/*
+
+orange: #f5c85f
+gray-bg: #EEEEEE
+dark-gray text: #111111
+gray text: #999999
+
+Font: Open Sans
+
+*/
+
+html {
+  font-family: Open Sans;
+}
+
+body {
+  min-height: 100%;
+  margin: 0;
+  padding-top: 30px;
+
+  /* Centrado de elementos */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  /* --------------------*/
+
+  /* Borde superior naranja */
+  border-top-style: solid;
+  border-top-width: 6px;
+  border-top-color: #f5c85f
+  /* ------------------- */
+}
+
+hr {
+  width: 3px;
+  height: 300px;
+  margin: 25px;
+  border: none;
+  background: linear-gradient(rgba(255, 193, 7, 0), #FF9800, rgba(0, 128, 0, 0));
```

Comment:
i think its better to use:  ```linear-gradient(rgba(255, 193, 7, 0), #FF9800, rgba(255, 193, 7, 0))``` as it will give the idea of starting and end in the same color. The last one may be confusing.